src/interface/meta.thrift

```diff
@@ -352,6 +373,70 @@ struct HBReq {
     1: common.HostAddr host,
 }
 
+struct CreateTagIndexReq {
+    1: common.GraphSpaceID space_id,
+    2: string              index_name,
+    3: TagIndexProperties  properties,
+}
+
+struct DropTagIndexReq {
+    1: common.GraphSpaceID space_id,
+    2: string              index_name,
+}
+
+struct GetTagIndexReq {
+    1: common.GraphSpaceID space_id,
+    2: string              index_name,
```

Comment:
Is IndexID better？

Reply:
IMO, it should be same with `GetTagReq`

Reply:
Aha!  This interface will also be used for the storage layer . Because the IndexID was saved in the index key. So I'm wondering if IndexID is better?